protocols/raft/src/main/java/io/atomix/protocols/raft/roles/LeaderAppender.java

```diff
@@ -291,7 +298,7 @@ protected void handleAppendResponseOk(
     // If we've received a greater term, update the term and transition back to follower.
     else if (response.term() > raft.getTerm()) {
       log.info(
-          "Received higher term ({} > {}) from {}",
+          "Received successful append response higher term ({} > {}) from {}, stepping down",
```

Comment:
Nit: It's not "stepping down" but transitioning to follower because there is a new leader.

Reply:
Aren't we stepping down if we're the leader? :thinking: Is stepping down terminology exclusively for when the leader abdicates without knowing the next leader?

I'm not arguing, just want to know :sweat_smile: I'll reword it :+1: